src/test/java/CanisLD/MockedLoans.java

```diff
@@ -0,0 +1,84 @@
+package CanisLD;
+
+import java.math.BigDecimal;
+import java.math.RoundingMode;
+import java.util.List;
+
+public class MockedLoans {
+  public static final String LOAN_A = "Loan A";
+  public static final String LOAN_B = "Loan B";
+
+  public static final FinanceProjection.Loan SAMPLE_LOAN_A =
+    new FinanceProjection.Loan.Builder()
+    .label(LOAN_A)
+    .start(0L)
+    .end(69L)
+    .cost(4000.00)
+    .loanDetails(
+      new LoanAmortization.Loan.Builder()
+        .amount(500000.00)
+        .interestRate(0.035)
+        .paymentFrequency(12L)
+        .numberOfTerms(30L)
+        .build()
+    )
+    .build();
+  
+  public static final FinanceProjection.Loan SAMPLE_LOAN_B =
+    new FinanceProjection.Loan.Builder()
+    .label(LOAN_B)
+    .start(69L)
```

Comment:
Is there a simpler way to set up the request data. Do we expect the user to re-establish the starting payment index on the refinance, or the new amount (as an absolute, since we might not know what the balance was.)